### PR TITLE
Remove LIST_LAZY_PROPERTY_NAMES_ROUTINE_NAME macro function.

### PR DESCRIPTION
**Pull request by zherczeg**
8K binary save (164K -> 156K), a small slowdown (1%). I think 8K worth it.

### CODE REVIEW

File: jerry-core/ecma/builtin-objects/ecma-builtins.c [hunk 1]
```diff
@@ -270,6 +270,20 @@ ecma_finalize_builtins (void)
...
+{
+#define BUILTIN(builtin_id, \
+                object_type, \
+                object_prototype_builtin_id, \
+                is_extensible, \
+                is_static, \
+                lowercase_name) \
+  ecma_builtin_ ## lowercase_name ## _property_descriptor_list,
+#include "ecma-builtins.inc.h"
+};
```
Comment (LaszloLango) [inline jerry-core/ecma/builtin-objects/ecma-builtins.c]: Don't we need `#undef BUILTIN` here?

File: jerry-core/ecma/builtin-objects/ecma-builtins.c [hunk 2]
```diff
@@ -559,42 +553,75 @@ ecma_builtin_list_lazy_property_names (ecma_object_t *object_p, /**< a built-in
...
-      case builtin_id: \
-                       { \
-                         ecma_builtin_ ## lowercase_name ## _list_lazy_property_names (object_p, \
-                                                                                       separate_enumerable, \
-                                                                                       main_collection_p, \
-                                                                                       non_enum_collection_p); \
-                         return; \
-                       }
-#include "ecma-builtins.inc.h"
+      uint32_t bit;
```
Comment (LaszloLango) [inline jerry-core/ecma/builtin-objects/ecma-builtins.c]: what bit? :) Please add a better name.

File: jerry-core/ecma/builtin-objects/ecma-builtins.c [hunk 3]
```diff
@@ -559,42 +553,75 @@ ecma_builtin_list_lazy_property_names (ecma_object_t *object_p, /**< a built-in
...
+        }
+      }
+
+      bool is_existing = true;
+
+      ecma_string_t *name_p = ecma_get_magic_string (curr_property_p->magic_string_id);
+
+      if (was_instantiated)
+      {
+        if (ecma_op_object_get_own_property (object_p, name_p) == NULL)
```
Comment (LaszloLango) [inline jerry-core/ecma/builtin-objects/ecma-builtins.c]: Merge these two conditions:

```C
if (was_instantiated && ecma_op_object_get_own_property (object_p, name_p) == NULL)
{
  is_existing = false;
}
```

File: jerry-core/ecma/builtin-objects/ecma-builtins.c (hunk 1 above)
Comment (zherczeg) [inline jerry-core/ecma/builtin-objects/ecma-builtins.c]: That is part of ecma-builtins.inc.h

File: jerry-core/ecma/builtin-objects/ecma-builtins.c (hunk 2 above)
Comment (zherczeg) [inline jerry-core/ecma/builtin-objects/ecma-builtins.c]: It was in the original code and used multiple places. But I can change it.

File: jerry-core/ecma/builtin-objects/ecma-builtins.c (hunk 1 above)
Comment (LaszloLango) [inline jerry-core/ecma/builtin-objects/ecma-builtins.c]: OK

**Comment by zherczeg** [comment]
Patch updated.

**Comment by LaszloLango** [comment]
LGTM

**Comment by dbatyai** [comment]
LGTM

**Pull request merged**